packages/flame/lib/src/components/position_component.dart

```diff
@@ -332,6 +343,23 @@ class PositionComponent extends Component
   /// The absolute center of the component.
   Vector2 get absoluteCenter => absolutePositionOfAnchor(Anchor.center);
 
+  /// Returns the angle formed by component's forward vector and a vector
+  /// starting at component's absolute position and ending at target.
+  /// Uses [nativeAngle] to decide the forward direction of the component.
+  /// See [lookAt] to make the component instantly rotate towards target.
+  /// Note: Returns 0 if target is exactly same as [absolutePosition].
```

Comment:
> * How about "orientation vector"

Habitual of using forward vector because of other engines, but orientation vector sounds easier to understand. Will change this.


> * or is it in CCW?

No, it is CW. Will mention that in the comments as well.



> * The last sentence is inaccurate if the nativeAngle is non-zero. I think a more accurate statement would be that if a target coincides with the current component, then it is treated as being north.

Ohhh! Good catch. Will add a test for this case as well.

Reply:
Made the suggested changes.